### ISSUE TITLE
pen not recognized for 22hd driver

### ISSUE THREAD

**Issue by junkyarddesigns** [georgejecook/BostoTabletDriverMac#11]
hi there i have installed the main mac os 22hd tablet driver and your driver and it does not seem to recognize the pen on your driver main page. are these 2 drivers conflicting? i run osx 10.8 mountain lion and ps6 and sketchbook pro mainly. the pen works in both but is very inconsistant at best and starts heavy and this out. is there something i am missing or is the 22hd that different from the earlier models? i have reinstalled your driver a few times with and without the bosto mfg driver and still same problem. thanks so much for taking the time to make a driver for the mac. you would think it would be a high priority for bosto to have all the problems prehashed out in advance before their product launch for both mac and windows. thanks to guys like you these products can still be available.

**Comment by fredewis** [comment]
Well I have sold my bosto as they will never release a osx driver I waited like they said and nothing (6 to 8 month) as great as a kit it may seem if you didn't buy it for windows ...I'm afraid it will be the end of the road update wise ..all they care about seems to be to sell the product and that's it.

**Comment by junkyarddesigns** [comment]
that sucks. so what your saying is your journey on the driver development is over?
they seem to have made some promises that are failing to deliver on. glad i kept my box. thanks so much for your replies. leroy

**Comment by fredewis** [comment]
yea the main developer does not have the tablet anymore and he was the only one putting his time into making it better and work with osx ....and that was months ago so i sold mine and im saving for a cintiq as over-prize as they may be the drivers are always upgraded.

**Comment by junkyarddesigns** [comment]
Good plan. Go with the best and these problems don't happen. Thanks a bunch. Best of luck

Leroy Valentine at uniontattoo.net

**Comment by georgejecook** [comment]
Sorry guys, especially Freddy - I really dug your enthusiasm. I think ultimately you guys are right. The Bosto guys should spend a few dollars to get the os x driver sorted out - It would cost them honestly, less than 500 bucks to get this driver extended to cover all their tablets on mac. My advice has now changed to "Don't buy a bosto if you have mac".

**Comment by fredewis** [comment]
Yes ...is true and I want to thank you George for all the time you put for it to work as best as you could. On the positive side I got my self a 13hd cintiq and wow, I'm glad just wished I had saved up for it early on but at least I can compare 2 products and the old saying "you pay for what you get" applys here.
